app/presenters/publishing_api_presenters/html_attachment.rb

```diff
@@ -0,0 +1,68 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::HtmlAttachment < PublishingApiPresenters::Item
+  def links
+    {
+      parent: [
+        parent.content_id
+      ],
+      organisations: parent.organisations.map(&:content_id),
+    }
+  end
+
+private
+
+  def document_format
+    "html_publication"
+  end
+
+  def details
+    {
+      body: body,
+      headings: headings,
+      public_timestamp: public_timestamp,
+      first_published_version: first_published_version?
+    }
+  end
+
+
+  def base_path
+    item.url
+  end
+
+  def description
+    #not used in this format
+  end
+
+  def public_updated_at
+    item.updated_at
+  end
+
+  def body
+    Whitehall::GovspeakRenderer.new.govspeak_to_html(govspeak_content.body)
```

Comment:
I presume this doesn't currently keep the `:headers` option, which decides whether to create markup for the special HTML numbered sections, and that this feature will come later?